prowler/__main__.py

```diff
@@ -280,9 +282,34 @@ def prowler():
             )
             sys.exit(1)
 
+    # Outputs
+    # TODO: this part is needed since the checks generates a Check_Report_XXX and the output uses Finding
+    # This will be refactored for the outputs generate directly the Finding
+    finding_outputs = []
+    for finding in findings:
+        finding_outputs.append(
+            Finding.generate_output(
+                global_provider, finding, global_provider.output_options
```

Comment:
Could we pass just the `global_provider` here? I know we should pass less information to it but for now we need that.

Reply:
done